exs/mundo_2/ruby/054.rb

```diff
@@ -0,0 +1,23 @@
+=begin
+Desafio 054
+
+Problema: Crie um programa que leia o ano de nascimento
+          de sete pessoas. No final, mostre quantas pessoas
+          ainda não atingiram a maioridade e quantas já são maiores.
+
+Resolução do problemas:
+=end
+
+maiores = 0
+menores = 0
+7.times do |i|
+	print"Digite a idade da #{i+1} pessoa: "
+	idade = gets.chomp.to_i
+	if idade >= 18
+		maiores += 1
+	end
+	menores += 1
```

Comment:
Neste caso, você está fazendo `maiores++` quando a idade é maior ou igual a 18, além disso está fazendo menores++ em *todos* os casos.
```suggestion
	if idade >= 18
		maiores += 1
	else 
		menores += 1
	end
```